atomic_reactor/plugins/post_compare_components.py

```diff
@@ -87,6 +87,9 @@ def run(self):
         """
         Run the plugin.
         """
+        if self.workflow.builder.base_from_scratch:
+            self.log.info("comparing components is not supported for FROM-scratch images")
```

Comment:
We can be more specific about action: `Skipping comparing components: unsupported ....` because it is not clear from that message that plugin was skipped.

Reply:
ok